example/src/EthereumEventsButtons.tsx

```diff
@@ -0,0 +1,48 @@
+import { useWeb3React } from '@web3-react/core'
+
+export const EthereumEventsButtons = () => {
+  const { isActive, provider, chainId } = useWeb3React()
+
+  const sendTransaction = async () => {
+    if ([137, 1].includes(chainId!)) {
+      alert('Transaction is not executed. Change to testnet in order not to lose your money')
+      return
+    }
+    const signer = provider?.getSigner()
```

Comment:
I thought `getSigner()` returned a promise

Reply:
No, it returns just object